src/sample_data/valid/Entity-emsl-example.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Giving these the name "Entity-" should allow them to be used in the documentation... unfortunately that feature isn't working ATM but I am going to troubleshoot to see if I can get it to function.